### PR TITLE
fix: Don't in-line `Metrics` property on MetricsResponse so that it matches swagger

### PR DESCRIPTION
**Pull request by lenny-intel**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://wiki.edgexfoundry.org/display/FA/Contributor%27s+Guide
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Metrics property on MetricsResonse was in-lined, thus not matching the swagger definition.

Issue Number: #280

## What is the new behavior?

Metrics property on MetricsResonse was no longer in-lined, thus it now matches the swagger definition.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Are there any new imports or modules? If so, what are they used for and why?
No
## Other information

### CODE REVIEW

File: v2/dtos/common/metrics.go [hunk 1]
```diff
@@ -21,7 +21,7 @@ type Metrics struct {
 // https://app.swaggerhub.com/apis-docs/EdgeXFoundry1/core-data/2.x#/MetricsResponse
 type MetricsResponse struct {
 	Versionable `json:",inline"`
-	Metrics     `json:",inline"`
+	Metrics     Metrics `json:"metrics"`
```
Comment (jim-wang-intel) [inline v2/dtos/common/metrics.go:24]: Extra Metrics variable name in the struct? did not see it doing that way originally

Comment (lenny-intel) [reply v2/dtos/common/metrics.go:24]: Originally? This is new V2 DTO.

See swagger here: 
https://github.com/edgexfoundry/app-functions-sdk-go/blob/master/openapi/v2/app-functions-sdk.yaml#L64

Comment (jim-wang-intel) [reply v2/dtos/common/metrics.go:24]: ok

**Review by jim-wang-intel — APPROVED** [review]

**Review by cloudxxx8 — APPROVED** [review]
LGTM

**Pull request merged**